util/topgen.py

```diff
@@ -287,7 +287,6 @@ def generate_plic(top, out_path):
     with rtl_gen_path.open(mode='w', encoding='UTF-8') as fout:
         fout.write(genhdr + gencmd + out)
 
-
```

Comment:
I think flake8 requires two lines between the functions.

Reply:
huh... you're right. Why didn't this fail lint..?